core/common/modules/finder/categories/create.php

```diff
@@ -51,7 +51,13 @@ public function get_category_items( array $options = [] ) {
 				continue;
 			}
 
-			if ( Source_Local::CPT === $post_type ) {
+			if ( ! empty( $post_type_object->class ) ) {
+				/**
+				 * @var $class \Elementor\Core\Base\Document
+				 */
+				$class = $post_type_object->class;
+				$url = $class::get_new_create_url();
```

Comment:
Remove from here. if the logic is needed move it to `documents->get_create_new_post_url`